modules/failure-handler/src/main/java/org/apache/ignite/internal/failure/configuration/FailureProcessorConfigurationSchema.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.failure.configuration;
+
+import org.apache.ignite.configuration.annotation.ConfigValue;
+import org.apache.ignite.configuration.annotation.ConfigurationRoot;
+import org.apache.ignite.configuration.annotation.ConfigurationType;
+import org.apache.ignite.internal.failure.handlers.configuration.FailureHandlerConfigurationSchema;
+
+/**
+ * Failure processor configuration schema.
+ */
+@ConfigurationRoot(rootName = "failureProcessor", type = ConfigurationType.LOCAL)
```

Comment:
Why it's local? We wont different values on different nodes?

Reply:
IMHO, this policy is about local node behavior and does not relate to the whole cluster. This might be useful for testing purposes as well.
The approach is the same as we use to configure critical workers https://github.com/apache/ignite-3/blob/main/modules/workers/src/main/java/org/apache/ignite/internal/worker/configuration/CriticalWorkersConfigurationModule.java#L34

Reply:
What's the point in having different failure detector configuration (logic) on different nodes? I don't have strong position here, just wondering why it's local, and thus different.